libs/designsystem/src/lib/components/modal/services/modal.helper.ts

```diff
@@ -43,8 +44,11 @@ export class ModalHelper {
         config.flavor === 'drawer' ? 'kirby-drawer' : null,
         config.flavor === 'compact' ? 'kirby-modal-compact' : null,
         modalSize,
+        config.interactWithBackground ? 'interact-with-background' : null,
+        config.invertColors ? 'invert-colors' : null,
```

Comment:
As per tech refinement, this should be handled by exposing css custom props for background + text color and let the target project set those